src/install.rs

```diff
@@ -281,15 +325,102 @@ pub struct InstallPaths {
 }
 
 impl InstallPaths {
-    pub fn new(_name: &str, args: &ArgMatches, capi_config: &CApiConfig) -> Self {
+    fn is_debianlike(target: &Target) -> bool {
+        consts::ARCH.eq_ignore_ascii_case(&target.arch)
```

Comment:
to make it safer or less surprising that should trigger only if --target is not passed. (and needs to be documented)

Reply:
*nods* I'll implement an approach in a separate commit. Let me know if it looks OK.